go/vt/vtctl/workflow/server.go

```diff
@@ -1990,7 +1990,7 @@ func (s *Server) dropTargets(ctx context.Context, ts *trafficSwitcher, keepData,
 				return nil, err
 			}
 		case binlogdatapb.MigrationType_SHARDS:
-			if err := sw.dropTargetShards(ctx); err != nil {
+			if err := sw.removeTargetTables(ctx); err != nil {
```

Comment:
This doesn't quite work because `removeTargetTables` doesn't remove tables correctly for a reshard workflow. Trying to fix that up now.